### ISSUE TITLE
VAMC System (incl banners): adopt V1 and V3 components where components are not used

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#16952]
## Description
VAMC System pages need the following updates to components:

Example pages:

- [Boston VAMC](https://www.va.gov/boston-health-care/)
- [Minneapolis VAMC](https://www.va.gov/minneapolis-health-care/)

### Elements that are not using components that should be (Backlog)

1. Link
   - These are currently not using a component at all but they should be using [the v1 component](https://design.va.gov/components/link/) 
   - This would also apply anywhere a basic link is used
   - NOTE: Active links for CTA [is actually a separate ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15598)
2. ~Telephone~
   - ~These are currently not using a component at all but they should be using `aria-describedby` [v1 component](https://design.va.gov/components/telephone#aria-described-by)~
   - ~Exts should be parsed correctly in the component~
3. [Back to top component](https://design.va.gov/components/back-to-top)
   - Update 5/3: After determining that this component does not actually have built in intelligence to only show based on page height:
      -  This should still be added, the System pages are typically longer so having this on these pages will probably make sense more often than not. 
4. Banner
   - The banners are not using any component but should be updated to use [the v1 component](https://design.va.gov/components/banner/)
   - Note regarding implementation: VBA already uses the component and has [it's own file](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/includes/vba_facilities/banner.liquid). I don't know if it would be useful/efficient/possible to have VAMCs and VBAs use one file so if thee ever needed to be updated again it would only be in 1 place rather than 2? 
   - At the time of writing this, [Durham](https://www.va.gov/durham-health-care/) as 2 banners for testing but who knows if they will be there later
![Screenshot 2024-03-13 at 3 18 13 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/3d1278ca-9524-4142-9079-7627fbd155e3)
5. Related Links/Spotlight
   - There is a list of links that we are going to change to align with the DS. This is going to use [Related Links](https://design.va.gov/components/link/collection#related-links). Based on Storybook, I don't believe there is standard component markup for this (i.e. `<va-XXX>`) [Here is the annotated Figma design](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16952) for what this will look like:
 ![Screenshot 2024-03-22 at 1 54 08 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/9eb2077d-1501-4099-8f8a-a6956f2db05b)

## Acceptance Criteria
- [x] Link component has been added
- [x] Back to top component has been added
- [x] Banner component has been added
- [x] Related Links have been added
- [x] Update and run existing test(s).
- [x] If additional tests are needed, stub a ticket to note this for future work.
- [x] Requires accessibility review
- [x] Change managemen for Related links

**Comment by maxx1128** [comment]
I estimate this ticket at a **3** for moving a good amount of relatively simple elements over to components (need I even say it again?)

**Comment by laflannery** [comment]
I added an AC for change management for Related links - I spoke to @mmiddaugh and since there is a large-ish display change with the Related links update she felt that giving folks a heads up that this was happening would be good.

**Comment by laflannery** [comment]
Telephone moved to #16148

**Comment by jilladams** [comment]
Custom banners on this page may or may not be weird in the Typography update. (Was weird, then branch updated, now is not weird.) In any case, we need to adopt the component and that's how we'll plan to fix the weird padding, if it stays weird in prod. 
https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1715017468641/ffb7fbb561bbca436ad535ecbbf0b96815413c29?wid=0-1715017487021&sender=u0b235d03cbd64f7f93673243
